### ISSUE TITLE
Can't Upgrade to 0.7.13

### ISSUE THREAD

**Issue by zacnomore** [gridsome/gridsome#1018]
## Description

This site cannot upgrade from `0.7.12` to `0.7.13`. It gives the below error when trying to run `gridsome develop`

### Steps to reproduce
Checkout out this branch: https://github.com/zacnomore/portfolio/tree/broken-upgrade

```
yarn
yarn develop
```

### Expected result
Successful build, development site launches

### Actual result
When I run `gridsome develop` I get this:

```
These dependencies were not found:

* core-js/modules/es.array.concat in ./node_modules/gridsome/app/fetch.js, ./node_modules/gridsome/app/head.js and 2 others
* core-js/modules/es.array.fill in ./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/babel-loader/lib??ref--1-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_
modules/vue-loader/lib??vue-loader-options!./src/components/Waves.vue?vue&type=script&lang=js&
* core-js/modules/es.array.find-index in ./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/babel-loader/lib??ref--1-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!.
/node_modules/vue-loader/lib??vue-loader-options!./src/templates/BlogPost.vue?vue&type=script&lang=js&, ./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/babel-loader/li
b??ref--1-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/templates/ProjectPost.vue?vue&type=script&lang=js&
* core-js/modules/es.array.for-each in ./node_modules/gridsome/app/head.js, ./node_modules/gridsome/app/directives/image.js and 3 others
* core-js/modules/es.array.from in ./node_modules/gridsome/app/directives/image.js
* core-js/modules/es.array.index-of in ./node_modules/gridsome/app/utils/class.js
* core-js/modules/es.array.is-array in ./node_modules/gridsome/app/router.js, ./node_modules/gridsome/app/utils/class.js
* core-js/modules/es.array.join in ./node_modules/gridsome/app/components/Image.js, ./node_modules/gridsome/app/utils/helpers.js
* core-js/modules/es.array.map in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/router.js and 1 other
* core-js/modules/es.array.slice in ./node_modules/gridsome/app/components/Image.js, ./node_modules/gridsome/app/utils/helpers.js
* core-js/modules/es.function.name in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/fetch.js and 1 other
* core-js/modules/es.number.constructor in ./node_modules/gridsome/app/components/Link.js
* core-js/modules/es.object.to-string in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/app.js and 4 others
* core-js/modules/es.parse-int in ./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/babel-loader/lib??ref--1-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_m
odules/vue-loader/lib??vue-loader-options!./src/components/Waves.vue?vue&type=script&lang=js&
* core-js/modules/es.regexp.constructor in ./node_modules/gridsome/app/components/Link.js, ./node_modules/gridsome/app/utils/helpers.js
* core-js/modules/es.regexp.exec in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/components/Link.js and 3 others
* core-js/modules/es.regexp.to-string in ./node_modules/gridsome/app/components/Link.js, ./node_modules/gridsome/app/utils/helpers.js
* core-js/modules/es.string.ends-with in ./node_modules/gridsome/app/directives/image.js
* core-js/modules/es.string.iterator in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/app.js and 1 other
* core-js/modules/es.string.link in ./node_modules/gridsome/app/head.js
* core-js/modules/es.string.replace in ./node_modules/gridsome/app/utils/helpers.js, ./node_modules/gridsome/app/utils/class.js
* core-js/modules/es.string.search in ./node_modules/gridsome/app/entry.client.js
* core-js/modules/es.string.split in ./node_modules/gridsome/app/utils/helpers.js, ./node_modules/gridsome/app/utils/class.js and 1 other
* core-js/modules/es.string.starts-with in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/components/Link.js and 1 other
* core-js/modules/es.string.trim in ./node_modules/gridsome/app/utils/class.js
* core-js/modules/es.symbol in ./node_modules/gridsome/app/app.js
* core-js/modules/es.symbol.description in ./node_modules/gridsome/app/app.js
* core-js/modules/es.symbol.iterator in ./node_modules/gridsome/app/app.js
* core-js/modules/web.dom-collections.for-each in ./node_modules/gridsome/app/head.js, ./node_modules/gridsome/app/directives/image.js and 3 others
* core-js/modules/web.dom-collections.iterator in ./node_modules/gridsome/app/entry.client.js, ./node_modules/gridsome/app/app.js

To install them, you can run: npm install --save core-js/modules/es.array.concat core-js/modules/es.array.fill core-js/modules/es.array.find-index core-js/modules/es.array.for-each core-js/modules/es.array.from core-js/modules/es.array.index-of core-js/modules/es.array.is-array core-js/modules/es.array.join core-js/modules/es.array.map core-js/modules/es.array.slice core-js/modules/es.function.name core-js/modules/es.number.constructor core-js/modules/es.object.to-string core-js/modules/es.parse-int core-js/modules/es.regexp.constructor core-js/modules/es.regexp.exec core-js/modules/es.regexp.to-string core-js/modules/es.string.ends-with core-js/modules/es.string.iterator core-js/modules/es.string.link core-js/modules/es.string.replace core-js/modules/es.string.search core-js/modules/es.string.split core-js/modules/es.string.starts-with core-js/modules/es.string.trim core-js/modules/es.symbol core-js/modules/es.symbol.description core-js/modules/es.symbol.iterator core-js/modules/web.dom-collections.for-each core-js/modules/web.dom-collections.iterator
```

### Environment
```
  System:
    OS: macOS 10.15.3
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  Binaries:
    Node: 13.8.0 - /usr/local/bin/node
    Yarn: 1.22.0 - /usr/local/bin/yarn
    npm: 6.13.7 - /usr/local/bin/npm
  Browsers:
    Chrome: 80.0.3987.122
    Safari: 13.0.5
  npmPackages:
    @gridsome/plugin-sitemap: ^0.2.3 => 0.2.3 
    @gridsome/remark-prismjs: ^0.3.0 => 0.3.0 
    @gridsome/source-filesystem: ^0.6.2 => 0.6.2 
    @gridsome/transformer-remark: ^0.5.0 => 0.5.0 
    gridsome: ^0.7.12 => 0.7.13 
  npmGlobalPackages:
    @gridsome/cli: 0.3.1
```

Using the following plugins:

```
    "@gridsome/plugin-sitemap": "^0.2.3",
    "@gridsome/remark-prismjs": "^0.3.0",
    "@gridsome/source-filesystem": "^0.6.2",
    "@gridsome/transformer-remark": "^0.5.0",
```

**Comment by BernardZhao** [comment]
I'm getting the same error, it seems to work by manually install corejs3. Is there a proper fix that I am missing?

**Comment by zacnomore** [comment]
I attempted to do that and got fewer errors but it still refused to build. I'm trying to dig deeper but would love additional insight. My current understanding is that babel removed its own polyfill system in favor of a decoupled one, core-js. 
Where's I'm lost is I would expect it to become a dependency of some kind for gridsome if it's required for development?

**Comment by noxify** [comment]
There is already an PR for the `core-js` problem.

https://github.com/gridsome/gridsome/pull/994

**Comment by oliverpool** [comment]
Which packages require `core-js` ? (run `yarn why core-js` to list them)

Does this [solution](https://github.com/oliverfindl/vue-svg-inline-loader/issues/39#issuecomment-587843573) work for you? (changing `vue-svg-inline-loader` with your dependency that requires core-js)


## Adding `core-js` as a dependency

I ran `yarn add "core-js@^3.6.4` and forced the `resolutions` in `package.json`:
```
"resolutions": {
    "vue-svg-inline-loader/core-js": "^2.6.11",
    "vue-svg-inline-loader/@babel/polyfill/core-js": "^2.6.11",
    "core-js": "^3.6.4"
  }
```
You pr

**Comment by zacnomore** [comment]
Yep, this worked great! Can you help me understand what's happening here? Is this causing all core-js dependencies to resolve as ^3.6.4?

**Comment by zacnomore** [comment]
Closed by #994
